Document/0x05a-Platform-Overview.md

```diff
@@ -512,8 +514,8 @@ The Android application attack surface consists of all components of the applica
   - [Testing for Sensitive functionality Exposure Through IPC](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x05h-Testing-Platform-Interaction.md#testing-for-sensitive-functionality-exposure-through-ipc "Testing for Sensitive functionality Exposure Through IPC");

```

Comment:
The "See" right before this line is missing a ":"

Reply:
done